src/pdf.sandbox.external.js

```diff
@@ -81,6 +81,11 @@ class SandboxSupportBase {
         ) {
           return;
         }
+
+        if (callbackId === 0) {
+          this.win.clearTimeout(this.timeoutIds.get(0));
+        }
```

Comment:
Should we have similar code in the `setInterval` method below?
If not, maybe add a (brief) comment to explain what's "special" about this case?